server/cli/start.go

```diff
@@ -230,6 +230,7 @@ func runQuit(cmd *cobra.Command, args []string) {
 var nodeCmds = []*cobra.Command{
 	initCmd,
 	startCmd,
+	certCmd,
```

Comment:
Good point. I originally placed it here to get the ordering of commands that I wanted. I've switched to manually adding all of the individual commands (in the desired order) in a single call to `cockroachCmd.AddCommand`.